### DISCUSSION TITLE
Window is not rename when you call `rename-window`

### DISCUSSION THREAD

**Discussion by gozes** [catppuccin/tmux#401] [Help]
So I just started using this theme but I just notice that when you call `rename-window` it will still keep the name of the directory or the name of the application running in that window. is that correct? Shouldn’t it show the name I set instead of the directory?

**Comment by nekowinston** [comment]
Hello :wave:
Thanks for the issue, with the [update to v0.2.0](https://github.com/catppuccin/tmux/commit/49845e02b38c5070e95bf4083feaf886940db68b), this should not happen. Please feel free to re-open the issue, should it persist on your end.
![CleanShot 2022-08-30 at 13 01 27](https://user-images.githubusercontent.com/79978224/187420515-09a13d02-178e-44b1-b959-3cc17997c84e.gif)

**Comment by Act0r1** [comment]
It is still don't change window name

**Comment by nekowinston** [comment]
@Act0r1
> This issue was closed a year ago. Please consider opening a [new issue](https://github.com/catppuccin/tmux/issues/new/choose) instead of leaving a comment here.

It's unlikely that the current maintainers will see your comment.

**Comment by nzagorsky** [comment]
For anyone wondering fix is the following:
```
set -g @catppuccin_window_default_text "#W"
set -g @catppuccin_window_current_text "#W"
```

**Comment by easystartup-io** [comment]
> For anyone wondering fix is the following:
> 
> ```
> set -g @catppuccin_window_default_text "#W"
> set -g @catppuccin_window_current_text "#W"
> ```

Thanks this works perfectly.

**Comment by lucadibello** [comment]
> For anyone wondering fix is the following:
> 
> ```
> set -g @catppuccin_window_default_text "#W"
> set -g @catppuccin_window_current_text "#W"
> ```

Thanks for sharing. Now works perfectly.

**Comment by LucianCrainic** [comment]
> For anyone wondering fix is the following:
> 
> ```
> set -g @catppuccin_window_default_text "#W"
> set -g @catppuccin_window_current_text "#W"
> ```

By using this it does actually fix the problem of seeing the actual name of the window that i have set, but when i am switching to another window the name changes to the default one which is the name of my computer. Any one got this problem too?

NOTE: i am going to add a video that shows the problem as soon as i get home.

**Reply by dreknix** [reply]
Some variable names have changed in v2 (see #291). Use now the following:

```sh
set -g @catppuccin_window_text "#W"
set -g @catppuccin_window_current_text "#W"
```

**Reply by dyllan-p** [reply]
> Some variable names have changed in v2 (see #291). Use now the following:
> 
> ```shell
> set -g @catppuccin_window_text "#W"
> set -g @catppuccin_window_current_text "#W"
> ```

I like some whitespace before the window name:

```shell
set -g @catppuccin_window_text " #W"
set -g @catppuccin_window_current_text " #W"
```

**Comment by jchai01** [comment]
To ensure that the window name always stays as what you named it, be sure to add these 3 lines:
```
set -g @catppuccin_window_default_text "#W"
set -g @catppuccin_window_text "#W"
set -g @catppuccin_window_current_text "#W"
```

**Reply by kjnsn** [reply]
Note that `@catppuccin_window_default_text` has been removed and no longer does anything
